doc/modules/model_evaluation.rst

```diff
@@ -1618,7 +1618,7 @@ then the explained variance is estimated as follow:
 
 .. math::
 
-  \texttt{explained\_{}variance}(y, \hat{y}) = 1 - \frac{Var\{ y - \hat{y}\}}{Var\{y\}}
+  explained\_{}variance(y, \hat{y}) = 1 - \frac{Var\{ y - \hat{y}\}}{Var\{y\}}
 
```

Comment:
Remove "{}"?

Reply:
If I understand correctly, then yes, I do build the user_guide (pdf) locally and the html version (locally) as well.  Both provide expected rendering by simply removing the enveloping \text{} or 
\texttt{}.